src/uwtools/drivers/fv3.py

```diff
@@ -100,29 +100,29 @@ def field_table(self):
     @tasks
     def files_copied(self):
         """
-        Files copied for FV3 run.
+        Files copied for run.
```

Comment:
We're probably going to copy-paste some tasks between drivers, so I tried to make their docstrings less likely to have too-specific terms that we'll inevitably forget to fix. Of course, we should factor _identical_ tasks out where it makes sense (e.g. to the new `uwtools.utils.tasks` module), but my guess is that there will be some copy-paste-lightly-customize going on.